libnavigation-base/src/main/java/com/mapbox/navigation/base/trip/TripNotification.kt

```diff
@@ -3,6 +3,12 @@ package com.mapbox.navigation.base.trip
 import android.app.Notification
 import com.mapbox.navigation.base.trip.model.RouteProgress
 
+/**
+ * Defines notification showing to user during navigation process
+ * launched with a [TripSession].
+ *
+ * @since 1.0.0
```

Comment:
Same as everywhere else - possibly we can skip these.

Reply:
We can, but I think it is good to mark the version of SDK when this or some other classes/methods comes